webserver/cors/src/main/java/io/helidon/webserver/cors/CrossOriginConfig.java

```diff
@@ -218,6 +228,21 @@ public long maxAgeSeconds() {
         return maxAgeSeconds;
     }
 
+    /**
+     * Reports whether the specified HTTP method name matches this {@code CrossOriginConfig}.
+     *
+     * @param method HTTP method name to check
+     * @return true if this {@code CrossOriginConfig} matches the specified method; false otherwise
+     */
+    public boolean matches(String method) {
+        for (String allowMethod : allowMethods) {
+            if (allowMethod.equals(method) || allowMethod.equals("*")) {
```

Comment:
HTTP methods should be compared ignore case (unless you fix the requestedMethod and configured methods somewhere to be always upper/lower cased)